system/HTTP/IncomingRequest.php

```diff
@@ -721,7 +721,7 @@ protected function parseRequestURI(): string
 		$query = $parts['query'] ?? '';
 		$uri   = $parts['path'] ?? '';
 
-		if (isset($_SERVER['SCRIPT_NAME'][0]))
+		if (isset($_SERVER['SCRIPT_NAME'][0]) AND pathinfo($_SERVER['SCRIPT_NAME'], PATHINFO_EXTENSION) === 'php')
```

Comment:
use `&&` instead of 'AND'

```suggestion
              if (isset($_SERVER['SCRIPT_NAME'][0]) && pathinfo($_SERVER['SCRIPT_NAME'], PATHINFO_EXTENSION) === 'php')
```

Reply:
done